pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionUtils.java

```diff
@@ -180,6 +180,7 @@ public static boolean isFitForDictionaryBasedComputation(String functionName) {
     return functionName.equalsIgnoreCase(AggregationFunctionType.MIN.name())
         || functionName.equalsIgnoreCase(AggregationFunctionType.MAX.name())
         || functionName.equalsIgnoreCase(AggregationFunctionType.MINMAXRANGE.name())
+            || functionName.equalsIgnoreCase(AggregationFunctionType.DISTINCT.name())
```

Comment:
`DISTINCT` is not modeled as aggregation on server side. We have it as a aggregation function type for historical reason, but we should not reach here